x/vesting/spec/04_transactions.md

```diff
@@ -56,3 +56,20 @@ The msg content stateless validation fails if:
 
 - `FunderAddress` or `AccountAddress` are invalid
 - `DestAddress` is not empty and invalid
+
+## `UpdateVestingFunder`
+
+```go
+type MsgUpdateVestingFunder struct {
+	// funder_address is the current funder address of the ClawbackVestingAccount
+	FunderAddress string `protobuf:"bytes,1,opt,name=funder_address,json=funderAddress,proto3" json:"funder_address,omitempty"`
+	// funder_address is the new address to replace the existing funder_address
+	NewFunderAddress string `protobuf:"bytes,2,opt,name=new_funder_address,json=newFunderAddress,proto3" json:"new_funder_address,omitempty"`
+	// vesting_address is the address of the ClawbackVestingAccount being updated
+	VestingAddress string `protobuf:"bytes,3,opt,name=vesting_address,json=vestingAddress,proto3" json:"vesting_address,omitempty"`
+}
+```
+
+The msg content stateless validation fails if:
+
+- `FunderAddress`, `NewFunderAddress` and `VestingAddress` are invalid
```

Comment:
This will fail if any one of those three addresses is invalid right?
```suggestion
- `FunderAddress`, `NewFunderAddress` or `VestingAddress` are invalid
```